x-pack/plugins/fleet/server/routes/data_streams/handlers.ts

```diff
@@ -107,8 +107,19 @@ export const getListHandler: RequestHandler = async (context, request, response)
         []
       )
     );
+    // Ignore dashboards not found
+    const allDashboardSavedObjects = allDashboardSavedObjectsResponse.saved_objects.filter((so) => {
+      if (so.error) {
+        if (so.error.statusCode === 404) {
+          return false;
+        }
+        throw so.error;
+      }
```

Comment:
I wonder if we can just swallow all SO errors (not just 404), we don't want to block the UI for any SO issues. instead, maybe we can just log any errors `logger.warn()`

Reply:
I guess if we have another error than a 404, it will probably be an issue with the connection between ES and Kibana something that in my opinion should throw a server error and the user will have the resend the request or reload the page.